src/clj/witan/send/params.clj

```diff
@@ -128,19 +117,20 @@
 
 (defn beta-params-movers
   "calculates the rate of the likelihood of a need-setting transitioning for an academic year"
-  [valid-states valid-transitions transitions]
+  [valid-states valid-year-settings transitions]
   (let [academic-years (->> (map first valid-states)
                             (distinct)
                             (sort))
-        observations (reduce (fn [coll {:keys [academic-year-1 need-1 setting-1 setting-2]}]
-                               (if (or (= setting-1 c/non-send)
+        observations (reduce (fn [coll {:keys [academic-year-1 need-1 need-2 setting-1 setting-2]}]
+                               (if (or (= setting-1 c/non-send) ; leaver or joiner
                                        (= setting-2 c/non-send))
                                  coll
-                                 (if (not= setting-1 setting-2)
-                                   (update-in coll [academic-year-1 (s/join-need-setting need-1 setting-1) :alpha] m/some+ 1)
-                                   (update-in coll [academic-year-1 (s/join-need-setting need-1 setting-1) :beta] m/some+ 1))))
+                                 (if (not (and (= setting-1 setting-2) ; not a remainer i.e a mover
+                                               (= need-1 need-2)))
```

Comment:
Is this logic definitely correct? It's late in the day so sorry, but should it not be 
```(if (and (not (= setting-1 setting-2)) (not (= need-1 need-2))))```?

Reply:
To be a stayer you have to have setting-1 = setting-2 *and* have need-1 = need-2.  Everything else (it's negation) is a mover.

By DeMorgan's law your logic is not equivalent to this (the equivalent would be setting-1 != setting-2 *or* need-1 != need-2. i.e, at least one is different - which is what we want).

What you've suggested is that *both* have to be different which was the issue in the first place.

Reply:
"To be a stayer you have to have setting-1 = setting-2 and have need-1 = need-2. Everything else (it's negation) is a mover." yep makes total sense in the light of day